### PR TITLE
Fixed: Embed task video in task section on the Part 1 set 1 page of the Gitbook training resource

### PR DESCRIPTION
**Pull request by JaisonBinns**
**This pull request makes the following changes:**
* Fixes issue #33 

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

Ping @Ifycode/git-github-training-resource

### CODE REVIEW

File: doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md [hunk 1]
```diff
@@ -42,8 +42,9 @@ You will now learn to use everything you have learnt about in the learning video
 The review you will get after you are done with the task will be based on the instructions in the issue and everything treated in the learning videos you have watched so far. Watch the video below to get better understanding on the tasks :thumbsup:
 {% endhint %}
 
-Task video goes here. Videos not available yet. Check back on the 9th of October 2022.
-
+{% embed url="https://youtu.be/SNfWwGD7UuI" %}
+Part 1 set 1 tasks 1 and 2
```
Comment (Ifycode) [inline doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md:46]: Nicely done. Can you help change the caption to "Tasks 1 and 2 for Part 1 set 1" instead?
Sorry for the stress.

Just update your branch.

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md (hunk 1 above)
Comment (Ifycode) [reply doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md:46]: Also remove "#33" from your pull request title.

Comment (Ifycode) [reply doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md:46]: Also remove "#33" from your pull request title @JaisonBinns

Comment (JaisonBinns) [reply doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md]: Just updated my branch with the changed caption and removed the issue "#33" from the PR title.

Comment (Ifycode) [reply doc/part-1-github-ui/set-1-github-ui-learning-videos-and-tasks.md]: Looks good @JaisonBinns
Thank you for your contribution

**Review by Ifycode — APPROVED** [review]

**Comment by Ifycode** [comment]
@all-contributors add @JaisonBinns for doc

**Pull request merged**